### PR TITLE
Configure project

### PR DESCRIPTION
**Pull request by rko619619**
1. Add activeadmin
2. Add rspec_rails
3. Add standard
4. Add Lefthook
5. Add brakeman
6. Add bundler-audit
7. Add standard-rails
8. Add simplecov
9. Add devise
10. Add site_prism
11. Add erb_lint
12. Add factory_bot
13. Add pg
14. Add rubocop-rails

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -68,5 +75,12 @@ group :test do
   # Use system testing [https://guides.rubyonrails.org/testing.html#system-testing]
   gem "capybara"
   gem "selenium-webdriver"
-
+  gem "simplecov"
 end
+
+gem "devise"
```
Comment (Volosh1n) [inline Gemfile]: better move them to the top, before all the groups

**Review by Volosh1n — APPROVED** [review]

File: Gemfile [hunk 2]
```diff
@@ -47,10 +47,17 @@ gem "bootsnap", require: false
 
 # Use Active Storage variants [https://guides.rubyonrails.org/active_storage_overview.html#transforming-images]
 # gem "image_processing", "~> 1.2"
+gem "activeadmin"
 
 group :development, :test do
   # See https://guides.rubyonrails.org/debugging_rails_applications.html#debugging-with-the-debug-gem
-  gem "debug", platforms: %i[ mri mingw x64_mingw ]
+  gem "debug", platforms: %i[mri mingw x64_mingw]
+  gem "rspec-rails"
```
Comment (ar2em1s) [inline Gemfile:24]: Setup rspec please (add factory_bot and others utilities config and etc)

**Review by ar2em1s — CHANGES_REQUESTED** [review]

File: Gemfile (hunk 1 above)
Comment (rko619619) [reply Gemfile]: Fixed.

File: Gemfile (hunk 2 above)
Comment (rko619619) [reply Gemfile:24]: Add setup for rspec and change factory_bot for factory_bot_rails

**Review by Volosh1n — APPROVED** [review]

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWD: example
+      POSTGRES_HOST: db
+    ports:
+      - 3000:3000
+  db:
+    image: postgres:alpine3.15
+    environment:
+      POSTGRES_PASSWORD: example
```
Comment (Volosh1n) [inline docker-compose.yml:14]: please, hide it somehow (.env or Rails.credentials)

**Review by Volosh1n — APPROVED** [review]

File: .idea/.gitignore
Comment (ar2em1s) [inline .idea/.gitignore:1]: Add all non-project files to gitignore please

File: .rspec [hunk 4]
```diff
@@ -0,0 +1 @@
+--require spec_helper
```
Comment (ar2em1s) [inline .rspec]: `--require rails_helper`

It would load everything since rspec_helper is included into rails_helper

File: Gemfile [hunk 5]
```diff
@@ -3,70 +3,53 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
...
-
-# Use Kredis to get higher-level data types in Redis [https://github.com/rails/kredis]
-# gem "kredis"
-
-# Use Active Model has_secure_password [https://guides.rubyonrails.org/active_model_basics.html#securepassword]
-# gem "bcrypt", "~> 3.1.7"
-
-# Windows does not include zoneinfo files, so bundle the tzinfo-data gem
-gem "tzinfo-data", platforms: %i[ mingw mswin x64_mingw jruby ]
+gem "tzinfo-data", platforms: %i[mingw mswin x64_mingw jruby]
```
Comment (ar2em1s) [inline Gemfile]: Redundant gem, it's needed only if you are working on Windows. Remove please

File: spec/spec_helper.rb [hunk 6]
```diff
@@ -0,0 +1,38 @@
+require "simplecov"
```
Comment (ar2em1s) [inline spec/spec_helper.rb]: Move simplecov requires and config to support/config/simplecov.rb and require it here please

**Review by ar2em1s — APPROVED** [review]

File: docker-compose.yml (hunk 3 above)
Comment (rko619619) [reply docker-compose.yml:14]: Fixed.

File: .rspec (hunk 4 above)
Comment (rko619619) [reply .rspec]: Fixed.

File: .idea/.gitignore
Comment (rko619619) [reply .idea/.gitignore:1]: Fixed.

File: Gemfile (hunk 5 above)
Comment (rko619619) [reply Gemfile]: Fixed.

File: spec/spec_helper.rb (hunk 6 above)
Comment (rko619619) [reply spec/spec_helper.rb]: Fixed.

**Pull request merged**